testProjects/packageJsonConfig/handler.test.js

```diff
@@ -8,13 +10,28 @@ describe('Using package.json iopipe configuration', () => {
   });
 
   it('Has configuration', done => {
-    iopipe({ networkTimeout: 345 })((event, context) => {
+    let inspectableInvocation;
+    iopipe({
+      networkTimeout: 345,
+      plugins: [
+        inv => {
+          inspectableInvocation = inv;
+          return new MockPlugin(inv);
+        }
+      ]
+    })((event, context) => {
       try {
-        const { clientId, networkTimeout, plugins } = context.iopipe.config;
+        const { clientId, networkTimeout } = context.iopipe.config;
+        const { plugins } = inspectableInvocation;
+
         expect(clientId).toBe('package_json_config_token_wow');
         expect(networkTimeout).toBe(345);
-        expect(plugins.length).toBe(1);
-        expect(_.isFunction(plugins[0])).toBe(true);
+
+        expect(_.map(plugins, 'meta.name')).toEqual([
+          'mock-plugin',
+          '@iopipe/trace'
```

Comment:
This reads like you called the function with MockPlugin, so why is there @iopipe/trace?

Reply:
@pselle This particular testProject shows off the usage of `iopipe` inside of the package.json, where we define plugins to include `@iopipe/trace`. I believe the test to be accurate to the behavior we are looking for